dsl/procedures/createIngress/procedure.dsl

```diff
@@ -0,0 +1,13 @@
+import java.io.File
+
+procedure 'Create Ingress',
+    description: 'Configures default ingress network in Docker Swarm cluster', {
+    
```

Comment:
The procedure should include the 'setup' step to get the grape dependencies as usual.